docs/model_graph.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+from django.core.management import call_command
+
+
+def generate_model_graphs(app):
```

Comment:
Do you know if it's possible to exlude these mixin names? Ideally I'd just have the name of the model itself there (e.g. Betrokkene)

<img width="598" height="69" alt="afbeelding" src="https://github.com/user-attachments/assets/94a50c39-f3f2-439c-95f7-bf6b1184c532" />